enhancements/sig-policy/98-long-term-compliance-history/README.md

```diff
@@ -0,0 +1,455 @@
+# Long-term Compliance History of Policies
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+To address the gaps in policy compliance history, it is proposed to store compliance events in a central relational
+database so that they can be stored long-term and are queryable with rich metadata. Each managed cluster would be
+responsible for recording their compliance events through an HTTP API on the hub that abstracts the database.
+
+## Motivation
+
+The current policy compliance history is stored in the policy template's parent policy and is limited to just 10
+compliance events. When a policy is deleted, compliance history for that policy is lost for all clusters. If a cluster
+is deleted, all the compliance history for that cluster is lost. Additionally, the current compliance history is not
+easily queryable and does not provide rich context of the violation. One exception to this is that there are some
+metrics that expose whether a policy is compliant or noncompliant but those must be scraped by Prometheus and changes
+between scrapes will be lost, therefore, the data cannot be relied upon.
+
+When a user wants to pass an audit or determine when an incident occurred, it is required to have accurrate, historical,
+and queryable compliance data.
+
+### Goals
+
+1. Provide an accurate compliance history on the hub or other centralized location.
+1. The compliance history is stored in a queryable format.
+1. The complaince history can contain data for at least 2 years for a hub that manages 3000 managed clusters with 30
+   policies that mostly stay compliant.
+1. The compliance history retains data of deleted managed clusters and deleted policies.
+1. The outcome is that policy compliance events can be queried by a date range and at least the cluster name, parent
+   policy name, policy name, compliance state, and compliance message can be shown. The solution should allow extending
+   with more data.
+1. This feature should be opt-in.
+
+### Non-Goals
+
+1. Open Cluster Management will not manage the Postgresql database. The user must bring their own.
```

Comment:
It's not clear that only one hub can write data to this database.  Could a customer with multiple hubs use the same database?  If so maybe the schema needs a hub reference too.

Reply:
This schema allows the same database to be used for multiple Open Cluster Management hubs, though if managed cluster names overlap, it'd be less confusing if a separate database on the same Postgresql database be used.

I'll add this in the document.